### DISCUSSION TITLE
Ασκήσεις Προγραμματισμού

### DISCUSSION THREAD

**Discussion by p15bako** [courses-ionio/hci#1580] [Announcements]
Παρατήρησα πως στα projects υπάρχουν διαθέσιμες και ασκήσεις προγραμματισμού με κωδικό HCI. Αυτές ισχύουν ή να κάνουμε μόνο ασκήσεις γραμμές εντολών;

**Comment by c15pati** [comment]
Αν ακολουθήσεις τις ασκήσεις που υποδικνύονται στην [σελίδα του μαθήματος](https://github.com/courses-ionio/hci) υποθέτω θα βρεις ποιες ασκήσεις είναι να κάνεις. Δεν είμαι σίγουρος για τι πράγμα ακριβώς μιλάς, θα ήθελες να γίνεις πιο συγκεκριμένος;

**Reply by p15bako** [reply]
https://github.com/courses-ionio/projects Σε αυτό το λινκ, υπάρχουν όλα τα διαθέσιμα project. Υπάρχουν όμως και ασκήσεις προγραμματισμού που δεν αναφέρονται στον πίνακα με τα παραδοτέα, ούτε στα κριτήρια βαθμολογίας. Η ερώτηση μου λοιπόν είναι: Αυτές οι ασκήσεις προγραμματισμού είναι διαθέσιμες;

**Reply by c15pati** [reply]
Εφόσον τα κριτήρια βαθμολογίας είναι αυτά: 
Κριτήρια βαθμολογίας: 10% εβδομαδιαίο κουίζ + 10% ομαδικότητα και συνεργασία + 30% σύγχρονη συμμετοχή στην τάξη και ατομικές παρουσιάσεις προόδου + 20% Συμμετοχικό εκπαιδευτικό υλικό + 30% 2 ασκήσεις γραμμής εντολών + 10% προαιρετικό bonus
Θα έλεγα ότι οι συγκεκριμένες ασκήσεις είναι διαθέσιμες μόνο αν κάποιος θέλει να ασχοληθεί για προσωπικούς του λόγους με τον προγραμματισμό. Όχι για το μάθημα, προφανώς. Ελπίζω να είναι ξεκάθαρο αυτό.
